ci/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Terraform
+.terraform/
+terraform*
```

Comment:
can you ensure there are newlines at end of files please

Reply:
Added newline.